src/auth.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta, timezone
+from typing import Dict, Union
+
+from jose import jwt
+from passlib.context import CryptContext
+
+from src.config import Settings
+
+
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+
+def encode_access_token(data: dict, expires_delta: timedelta) -> dict:
```

Comment:
```suggestion
def encode_access_token(data: dict, expires_delta: timedelta) -> dict[str, str]:
```

위처럼 key, value 에도 타입을 명시하는게 좋습니다.

Reply:
네. 알겠습니다.